Editor/Scripts/BotManagement/RGBotSynchronizer.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.IO;
+using System.Text;
+using RegressionGames;
+using RegressionGames.Types;
+using UnityEditor;
+using UnityEngine;
+using File = UnityEngine.Windows.File;
+
+namespace Editor.Scripts.BotManagement
+{
+    public class RGBotSynchronizer
+    {
+        private static readonly RGBotSynchronizer _this = new ();
+
+        // This must match RGBotRuntimeManagement.cs
+        public static readonly string BOTS_PATH = "Assets/RegressionGames/Runtime/Bots";
+        
+        [MenuItem("Regression Games/Create New Bot")]
+        private static void CreateNewBot()
+        {
+            // create a new bot folder
+            var folderName = _this.CreateBotFolder("NewRGBot", out var botId);
+
+            // create the assets
+            _this.CreateNewBotAssets(folderName, "NewRGBot", botId);
+        }
+        
+        //[MenuItem("Regression Games/Synchronize Bots with RG")]
+        //TODO (REG-1306): Implement me
```

Comment:
We might want to add this as an experiment - we assume they want to sync bots with RG, but do they actually?

Reply:
My hypothesis is that for their own bots that they share across games, and for bots that we provide as a starting point, then yes? But I don't know!

Reply:
ok, i already had this broken out as a separate task.  I'll move it to next cycle for further discussion as I was working on getting the overlay correct first anyway :D

Reply:
Note added to Linear task here https://linear.app/regression-games/issue/REG-1306/sync-unity-local-bots-with-rg